File: TransCoder_tokenized_test_set_functions/python/SUBARRAYSUBSTRING_VS_SUBSEQUENCE_AND_PROGRAMS_TO_GENERATE_THEM_1.py
def printSubsequences ( arr, n ) :
    opsize = math . pow ( 2, n )
    for counter in range ( 1, ( int ) ( opsize ) ) :
        for j in range ( 0, n ) :
            if ( counter & ( 1 << j ) ) :
                print ( arr [ j ], end = " " )
        print ( )
    

File: TransCoder_tokenized_test_set_functions/java/SUBARRAYSUBSTRING_VS_SUBSEQUENCE_AND_PROGRAMS_TO_GENERATE_THEM_1.java
static void printSubsequences ( int n ) {
  int opsize = ( int ) Math . pow ( 2, n );
  for ( int counter = 1;
  counter < opsize;
  counter ++ ) {
    for ( int j = 0;
    j < n;
    j ++ ) {
      if ( BigInteger . valueOf ( counter ) . testBit ( j ) ) System . out . print ( arr [ j ] + " " );
    }
    System . out . println ( );
  }
}